src/app/(admin)/bundles/components/ExportLinks.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Link from "next/link";
+import { Bundle } from "@prisma/client"
+
+import { languages } from 'src/constants/languages';
+
+const generateLegacyOSEPublicBundleLink = (bundle: Bundle, language: string): string | null => {
+  const importMetadata = bundle.importMetadata as { grade?: string, unit?: string };
+  const grade = importMetadata.grade;
+  const unit = importMetadata.unit;
+
+  if (!grade || !unit) {
+    return null;
+  }
+  return `/public-bundles/from-import/${bundle.importSourceId}/grade%20${grade}/${unit}/${language}`;
+};
+
+interface ExportLinksProps {
+  bundle: Bundle;
+}
+
+const ExportLinks = ({ bundle }: ExportLinksProps) => {
+  const languageEntries = Object.entries(languages);
+
+  return (
+    <div className="export-links">
+      {languageEntries.map(([code, label], index) => {
+        const link = generateLegacyOSEPublicBundleLink(bundle, code)
+
+        if (!link) return null
```

Comment:
Let's try to check if there is data for this language in the OCX, and if not, return null